File: src/services/identity/Identity.DTO/AccountInfo.cs
namespace Identity.DTO
{
    using Identity.Ids;
    using Identity.ValueObjects;

    using MedEasy.Ids;
    using MedEasy.RestObjects;

    using System.Collections.Generic;

    public class AccountInfo : Resource<AccountId>
    {
        public UserName Username { get; set; }

        public Email Email { get; set; }

        public bool Locked { get; set; }

        /// <summary>
        /// Name associated with the account
        /// </summary>
        public string Name { get; set; }

        public TenantId TenantId { get; set; }

        public IEnumerable<ClaimInfo> Claims { get; set; }

        public IEnumerable<RoleInfo> Roles { get; set; }

        public AccountInfo() => Claims = new List<ClaimInfo>();
    }
}

File: src/clients/MedEasy.SPA/ClientApp/components/accounts/AccountInfo.ts

export class AccountInfo implements MedEasy.DTO.Resource<string> {

    username: string;
    email: string;
    locked: boolean;
    name: string;
    tenantId: string;
    claims: Array<{ type: string, value: string }>;
    id: string
    updatedDate?: Date;
    createdDate: Date;

}